packages/cli/src/commands/versions/bump.ts

```diff
@@ -158,6 +170,26 @@ export default async () => {
   }
 };
 
+function createVersionFinder() {
+  const found = new Map<string, string>();
```

Comment:
Should probably make the values `Promise<string>` and store the fetch.then(etc) in it so you don't get duplicate overlapping fetches (you currently only set the map after the await...)

Reply:
It's just deduplicating the info fetches between the dependencies and lockfile entry loops, not within them. Parallelizing those would be a separate refactor which I skipped since it's quick enough for now.